include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -621,9 +624,148 @@ struct SPhysicalDeviceFeatures
+    /* VK_EXT_video_decode_h264 */
+    /* VK_EXT_video_decode_h265 */
+    /* VK_EXT_video_encode_h264 */
+    /* VK_EXTX_portability_subset */
+    /* VK_GOOGLE_decorate_string */
+    /* VK_GOOGLE_display_timing */
+    /* VK_GOOGLE_hlsl_functionality1 */
+    /* VK_GOOGLE_sampler_filtering_precision */
+    /* VK_GOOGLE_user_type */
+    /* VK_HUAWEI_prerotation */
+    /* VK_HUAWEI_smart_cache */
+    /* VK_IMG_filter_cubic */
+
     // [DO NOT EXPOSE] Enables certain formats in Vulkan, we just enable them if available or else we need to make format support query functions in LogicalDevice as well
     /* VK_IMG_format_pvrtc */
-   
+
+    /* VK_INTEL_performance_query */
+    /* VK_KHR_bind_memory2 */
+    /* VK_KHR_copy_commands2 */
+    /* VK_KHR_create_renderpass2 */
+    /* VK_KHR_deferred_host_operations */
+    /* VK_KHR_descriptor_update_template */
+    /* VK_KHR_device_group */
+    /* VK_KHR_display_swapchain */
+    /* VK_KHR_external_fence */
+    /* VK_KHR_external_fence_fd */
+    /* VK_KHR_external_fence_win32 */
+    /* VK_KHR_external_memory */
+    /* VK_KHR_external_memory_fd */
+    /* VK_KHR_external_memory_win32 */
+    /* VK_KHR_external_semaphore */
+    /* VK_KHR_external_semaphore_fd */
+    /* VK_KHR_external_semaphore_win32 */
+    /* VK_KHR_format_feature_flags2 */
+    /* VK_KHR_get_memory_requirements2 */
+    /* VK_KHR_get_physical_device_properties2 */
+    /* VK_KHR_image_format_list */
+    /* VK_KHR_incremental_present */
+    /* VK_KHR_pipeline_library */
+    /* VK_KHR_relaxed_block_layout */
+    /* VK_KHR_shader_clock */
+    /* VK_KHR_shader_non_semantic_info */
+    /* VK_KHR_shared_presentable_image */
+    /* VK_KHR_storage_buffer_storage_class */
+    /* VK_KHR_surface */
+
+    /* VK_KHR_swapchain */ // we want to expose this extension as feature
+    bool swapchain = false;
+
+    /* VK_KHR_swapchain_mutable_format */
+    /* VK_KHR_video_decode_queue */
+    /* VK_KHR_video_encode_queue */
+    /* VK_KHR_win32_keyed_mutex */
+    /* VK_KHR_win32_surface */
+    /* VK_MESA_query_timestamp */
+    /* VK_MVK_ios_surface */
+    /* VK_MVK_macos_surface */
+    /* VK_MVK_moltenvk */
+    /* VK_NV_acquire_winrt_display */
+    /* VK_NV_clip_space_w_scaling */
+    /* VK_NV_cuda_kernel_launch */
+    /* VK_NV_dedicated_allocation */
+    /* VK_NV_device_debug_checkpoints */
+    /* VK_NV_external_memory_win32 */
+    /* VK_NV_fill_rectangle */
+    /* VK_NV_fragment_coverage_to_color */
+    /* VK_NV_framebuffer_mixed_samples */
+    /* VK_NV_geometry_shader_passthrough */
+    /* VK_NV_glsl_shader */
+    /* VK_NV_low_latency */
+    /* VK_NV_present_barrier */
+    /* VK_NV_rdma_memory */
+    /* VK_NV_sample_mask_override_coverage */
+    /* VK_NV_shader_subgroup_partitioned */
+    /* VK_NV_texture_dirty_tile_map */
+    /* VK_NV_video_queue */
+    /* VK_NV_viewport_array2 */
+    /* VK_NV_viewport_swizzle */
+    /* VK_NV_win32_keyed_mutex */
+    /* VK_QCOM_render_pass_shader_resolve */
```

Comment:
seems like a more limited and weird version of the `VK_AMD_shader_fragment_mask` lets wait on them to converge onto something useful